mongo/collection.go

```diff
@@ -1329,7 +1329,7 @@ func (coll *Collection) Find(ctx context.Context, filter interface{},
 		}
 		op.Sort(sort)
 	}
-	retry := driver.RetryNone
+	retry := driver.RetryTimeout
 	if coll.client.retryReads {
 		retry = driver.RetryOncePerCommand
```

Comment:
It seems like this will enable retrying reads when `timeoutMS` is set, even if `retryReads == false`. Is that correct? How does a user disable retrying reads when `timeoutMS` is set?